questions/javascript-questions.md

```diff
@@ -64,5 +64,5 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are the benefits of using `spread syntax` and how is it different from `rest syntax`?
 * How can you share code between files?
 * Why you might want to create static class members?
-* How to implement performance improvement of a search functionality, by executing search logic only when user finished typing?
-* How to restrict a function to only execute with a gap of atleast t seconds?
+* How to implement performance improvement of a search functionality, by executing search logic only when user finished typing? (debouncing)
```

Comment:
My previous feedback still stands. You have to make this question more generic. It's not a question about  search, it's a question about debouncing which can be applied to many different cases. coupling the question tightly to some specific use case might throw off someone unfairly. If you want to add a specific coding challenge, that's one thing. But testing for concepts has to be as **generic** as possible.  

I still think that a single, simple question that asks about both concepts is better, anyway. I would prefer that.